flink-libraries/flink-table/pom.xml

```diff
@@ -92,7 +92,12 @@ under the License.
 			</exclusions>
 		</dependency>
 
-
+		<dependency>
```

Comment:
Yes, this problem bothers me, too. the license of reflections jar is  WTFPL, does that means we would do anything, how to check whether it is compatible with AL2? I noticed that the reflections jar is already referenced in  flink-parent.pom, but only for tests, and would not be included in any flink jars. My original thought is not only use this jar, but also include it (like calcit) in the flink-table-{version}.jar.